.zenodo.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "CADET-Process: Version 0.9.2",
```

Comment:
@hannahlanzrath is this already the file for the next version?

Reply:
ok, next version will be 0.10.0

Reply:
I will change, amend and force push if you're ok with that